tests/test_0001_html.py

```diff
@@ -441,6 +441,21 @@ def test_missing_start_tag(self):
         ):
             HTML('</span>')
 
+    def test_self_closing_tag_with_slash(self):
+        node = HTML('<div><img src="abc"/></div>')[0]
+        assert node.nodes[0].tag_name == 'img'
+        assert node.nodes[0].self_closing_tag is True
+
+    def test_self_closing_tag_without_slash(self):
+        node = HTML('<div><img src="abc"></div>')[0]
+        assert node.nodes[0].tag_name == 'img'
+        assert node.nodes[0].self_closing_tag is True
+
+    def test_non_self_closing_tag_with_slash(self):
+        node = HTML('<div><span/></div>')[0]
```

Comment:
Should this work? That's no valid html.

Reply:
I thought about that. But decided not to change current behavior.

`<div><span/></div>` indeed invalid html5.
But this `<svg xmlns="http://www.w3.org/2000/svg"><path/></svg>` is valid html.
There are [foreign elements](https://html.spec.whatwg.org/#foreign-elements) that can be self closing in html5.

So I decided not to add rules to determine if tag can be self closing.
We can do it in a separate PR, if necessary.

Reply:
What means "check this line"?

Reply:
@maratori I am sorry. ``handle_starttag`` now has a keyword argument ``self_closing_tag``. Don't we have to check the value of ``self_closing_tag`` in https://github.com/lona-web-org/lona/blob/master/lona/html/parsing.py#L137 ?

Reply:
It depends on what behavior we expect here
```py
        node = HTML('<div><span/></div>')[0]
        assert node.nodes[0].tag_name == 'span'
        assert node.nodes[0].self_closing_tag is False  # or True
```
I'd say that we should not.

Reply:
I think an argument to let ``self_closing_tag`` be true in this case would be that ``str(node)`` would generate the same (broken) HTML that was parsed.

Reply:
Fixed